packages/gatsby/src/bootstrap/index.ts

```diff
@@ -45,6 +46,13 @@ export async function bootstrap(
     ...(await initialize(bootstrapContext)),
   }
 
+  const workerPool = context.workerPool
+  const directory = slash(context.store.getState().program.directory)
+
+  await Promise.all(
+    workerPool.all.loadConfigAndPlugins({ siteDirectory: directory })
+  )
```

Comment:
Just a note - I'm not sure we actually need to `await Promise.all` here - main process can continue here and run `customizeSchema`, `sourceNodes` etc while workers are loading config.

We also don't even need to to await it before we tell workers to`buildSchema` because `workerPool` will internally queue those calls on it's own

Reply:
And same thing for `buildSchema` - we can just tell workers to do it and when we start calling `runQueries` it will be workerPool job to ensure workers finished previous task before it will start running queries. I would think for PQR we only really need to await actualy query running execution batches